NBXplorer.Client/ExplorerClient.cs

```diff
@@ -322,6 +322,16 @@ public Task<GetBalanceResponse> GetBalanceAsync(DerivationStrategyBase userDeriv
 			return SendAsync<GetBalanceResponse>(HttpMethod.Get, null, "v1/cryptos/{0}/derivations/{1}/balance", new[] { CryptoCode, userDerivationScheme.ToString() }, cancellation);
 		}
 
+
+		public GetBalanceResponse GetBalance(string address)
```

Comment:
Should be a strongly typed `BitcoinAddress`, not a string.

Reply:
Ok

Reply:
Changed with new commit